cpp/tests/io/parquet_test.cpp

```diff
@@ -3269,7 +3269,6 @@ TEST_F(ParquetWriterTest, EmptyList)
 
   using lcw     = cudf::test::lists_column_wrapper<int64_t>;
   auto expected = lcw{lcw{}, lcw{}, lcw{}};
-  cudf::test::print(expected);
```

Comment:
I also recommend to change the line below (3273) into using `CUDF_TEST_EXPECT_COLUMNS_EQUAL`.

Reply:
Do we have a preference between `CUDF_TEST_EXPECT_COLUMNS_EQUAL` and `expect_columns_equal`?

By checking the repo, I do see `CUDF_TEST_EXPECT_COLUMNS_EQUAL` is more commonly used while `expect_columns_equal` is the legitimate utility recommended by [testing doc](https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/TESTING.md#column-comparison-utilities)

Reply:
Now I get why I keep finding `expect_columns_equal` functions in testing code. We should update the [testing doc](https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/TESTING.md#column-comparison-utilities).
The disadvantage of using `expect_columns_equal` is that, `expect_columns_equal` is a function, and inside it uses macros for testing individual properties of column. So, when a test of a property fails, it points to lines of `column_utilties.cu`, not the point of failure. 
When you use macro `CUDF_TEST_EXPECT_COLUMNS_EQUAL`, it gives the line of failure at the testing code. For eg, it will print `"Line 3273 cpp/tests/io/parquet_test.cpp <--  line of failure"`
That's why `CUDF_TEST_EXPECT_COLUMNS_EQUAL` is preferred.

Reply:
Exactly. Thanks for clarification :)

Reply:
Done